src/managers/PermissionManager.js

```diff
@@ -69,13 +70,12 @@ class PermissionManager {
         'administrator',
       );
 
-      const token = await this._generateAdminToken();
+      const aToken = await this._generateAdminToken();
```

Comment:
Cause ESLINT on my site says, don't use parameteobjects like

Token: Token

I will rename it to adminToken

Without a successfull LINT I can't commit ...:(